libs/aws/langchain_aws/logger_util.py

```diff
@@ -0,0 +1,27 @@
+import logging
+
+
+def get_logger(module_name: str) -> logging.Logger:
+    '''
+    Define a logger with the passed module_name at module level or function level
+    '''
+    logger = logging.getLogger(module_name)
+    logger.setLevel(logging.INFO)
+    return logger
+
+def configure_logger(logger: logging.Logger, log_level: int = None) -> logging.Logger:
+    '''
+    Define handler, formatter and custom logging level. Takes logger object and log_level as input and returns a logger object. We can modify this to have handler and formatter passed in as well.
+    '''
+    # define handler and formatter
+    handler = logging.StreamHandler()
+    formatter = logging.Formatter("%(asctime)s - %(levelname)s - %(name)s - %(message)s")
```

Comment:
Suggestion, move these as constants outside. You could let the user pass these formatters if you want. 

What does the rest of the `langchain` code use for logging?